resources/infix-query-grammar.ebnf

```diff
@@ -1,14 +1,16 @@
 (* -*- mode: bison -*- *)
 
-expr ::= eq | fn-call | ipv4-address | identifier
+expr ::= eq | fn-call | ipv4-addr | identifier
 
 identifier ::= #"[A-Za-z-]+"
-maybe-whitespace ::= " "*
+
+<sp-char> ::= " "
+<sp> ::= sp-char*
 
 fn-call ::= identifier <"("> identifier <")">
 
-eq ::= eqable <maybe-whitespace> <"="> <maybe-whitespace> eqable
-<eqable> ::= fn-call | identifier | ipv4-address
+eq ::= eqable ( <sp> <"="> <sp> eqable )+
```

Comment:
The real meat and potatoes here is just the `( ... ) +`; everything else is just shortening stuff.